### PR TITLE
Bluntsord/user guide introduction

### PR DESCRIPTION
**Pull request by Bluntsord**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -9,19 +9,36 @@ title: User Guide
...
 
+Are you a new user who is looking to get started with using DonnaFin? Perhaps you are a more 
+advanced user trying to refresh your understanding of the available commands? Look no further!
+This is the guide that will kickstart your journey as a DonnaFin whizz and even be your go-to
+reference if you need to refresh your understanding on some features. 
+Head over to [Quick Start](#2-quick-start) or [Features](#4-features) for more information.
+
 ### 1.2 How to use the user guide
 
+As most applications in the market are not CLI-based, this may come across
```
Comment (sheehui) [inline docs/UserGuide.md:33]: "on the market"

File: docs/UserGuide.md [hunk 2]
```diff
@@ -9,19 +9,36 @@ title: User Guide
...
+Are you a new user who is looking to get started with using DonnaFin? Perhaps you are a more 
+advanced user trying to refresh your understanding of the available commands? Look no further!
+This is the guide that will kickstart your journey as a DonnaFin whizz and even be your go-to
+reference if you need to refresh your understanding on some features. 
+Head over to [Quick Start](#2-quick-start) or [Features](#4-features) for more information.
+
 ### 1.2 How to use the user guide
 
+As most applications in the market are not CLI-based, this may come across 
+as new to some users. We strongly recommend that first time users read through the user guide to gain a basic understanding
```
Comment (sheehui) [inline docs/UserGuide.md:34]: first-time? or first time

File: docs/UserGuide.md [hunk 3]
```diff
@@ -9,19 +9,36 @@ title: User Guide
...
+of how to use our app. We have also included links to navigate around the user guide to allow easier access
+of related information. Some tips and tricks, as well as cautions are included throughout the guide to
+help you get more out of the application and avoid certain common dangers. Lastly, please do use the table of contents
```
Comment (sheehui) [inline docs/UserGuide.md:37]: "our application". 
"Lastly, please note that you may use the table of contents"
dangers sounds a bit funny ngl.

**Review by sheehui — COMMENTED** [review]
Great work on the intro @Bluntsord here's some small changes/nitpicking I have suggested

File: docs/UserGuide.md [hunk 4]
```diff
@@ -546,3 +546,15 @@ This section gives a quick summary of how you may use DonnaFin.
...
 | **Add Policy**   | `append n/POLICY_NAME i/INSURER iv/$INSURED_VALUE pr/$YEARLY_PREMIUM c/$COMMISSION`| `append n/Diamond Policy i/AIA iv/$10000 pr/$200 c/$1000`
 | **Remove Asset/Liability/Policy**   | `remove INDEX`                                                                 | `remove 1`
+
+--------------------------------------------------------------------------------------------------------------------
+
+## 7. Glossary
+
+* **CLI**: Abbreviation for Command line interface, where commands are processed in the form of text.
+This is opposed to the standard graphical user interface that we are use to.
+* **Private client detail**: A client detail that is not meant to be shared with others
```
Comment (sheehui) [inline docs/UserGuide.md:556]: I am not sure where we have used this. Maybe a more appropriate one would be client details, which we typically refer to as assets, liabilities,notes,policies,contacts. you get what i mean. pls don't quote me verbatim when u do the ug tho AHAHHAHA phrase it better

File: docs/UserGuide.md [hunk 5]
```diff
@@ -529,3 +546,15 @@ This section gives a quick summary of how you may use DonnaFin.
 | **Add Liability**   | `append n/LIABILITY_NAME ty/LIABILITY_TYPE v/$LIABILITY_VALUE r/REMARKS_ON_LIABILITY`| `append n/Property debt with DBS ty/debt v/$100000 r/10% annual interest`
 | **Add Policy**   | `append n/POLICY_NAME i/INSURER iv/$INSURED_VALUE pr/$YEARLY_PREMIUM c/$COMMISSION`| `append n/Diamond Policy i/AIA iv/$10000 pr/$200 c/$1000`
 | **Remove Asset/Liability/Policy**   | `remove INDEX`                                                                 | `remove 1`
+
+--------------------------------------------------------------------------------------------------------------------
+
+## 7. Glossary
+
+* **CLI**: Abbreviation for Command line interface, where commands are processed in the form of text.
+This is opposed to the standard graphical user interface that we are use to.
```
Comment (sheehui) [inline docs/UserGuide.md:555]: "standard.....that you may be used to"

File: docs/UserGuide.md [hunk 6]
```diff
@@ -529,3 +546,15 @@ This section gives a quick summary of how you may use DonnaFin.
...
+* **Private client detail**: A client detail that is not meant to be shared with others
+* **Asset**: Things of present or future value owned by a client
```
Comment (sheehui) [inline docs/UserGuide.md:576]: fullstop haha

File: docs/UserGuide.md [hunk 7]
```diff
@@ -9,19 +9,39 @@ title: User Guide
...
 ### 1.2 How to use the user guide
 
+As most applications in the market are not CLI-based, this may come across 
+as new to some users. We strongly recommend that first time users read through the user guide to gain a basic understanding
+of how to use our app. We have also included links to navigate around the user guide to allow easier access
+of related information. Some tips and tricks, as well as cautions are included throughout the guide to
+help you get more out of the application and avoid certain common dangers. Lastly, please do use the table of contents
+at the top of the document for easier navigation throughout document. 
+
+Click here to return to the top:
```
Comment (sheehui) [inline docs/UserGuide.md]: I think right, normally it would be "click <a href> here </a> to return to the top"

File: docs/UserGuide.md [hunk 8]
```diff
@@ -37,9 +37,9 @@ of related information. Some tips and tricks, as well as cautions are included t
 help you get more out of the application and avoid certain common dangers. Lastly, please do use the table of contents
 at the top of the document for easier navigation throughout document. 
 
-Click here to return to the top:
+--------------------------------------------------------------------------------------------------------------------
+This marks the end of the section. Click here to return to the top:
```
Comment (sheehui) [inline docs/UserGuide.md:40]: Can be a bit more specific with the "section", so use Introduction/Quick Start etc.

File: docs/UserGuide.md [hunk 9]
```diff
@@ -30,12 +30,14 @@ Head over to [Quick Start](#2-quick-start) or [Features](#4-features) for more i
...
 
-As most applications in the market are not CLI-based, this may come across 
-as new to some users. We strongly recommend that first time users read through the user guide to gain a basic understanding
+As most applications on the market are not CLI-based, this may come across 
+as new to some users. We strongly recommend that first-time users read through the user guide to gain a basic understanding
 of how to use our app. We have also included links to navigate around the user guide to allow easier access
 of related information. Some tips and tricks, as well as cautions are included throughout the guide to
-help you get more out of the application and avoid certain common dangers. Lastly, please do use the table of contents
-at the top of the document for easier navigation throughout document. 
+help you get more out of our application and avoid certain common dangers. Also, links back to the top of the page
```
Comment (sheehui) [inline docs/UserGuide.md:37]: the links back to the top can put after the table of contents one. "Additionally, present at the end of each section are links that will bring you back to the start of our User Guide."

File: docs/UserGuide.md [hunk 10]
```diff
@@ -30,12 +30,14 @@ Head over to [Quick Start](#2-quick-start) or [Features](#4-features) for more i
...
+like below are present at the end of every section. To allow for easier navigation
```
Comment (sheehui) [inline docs/UserGuide.md:38]: For easier navigation around this User Guide, we suggest using our table of contents together with these links to quickly find and access the information you need.

File: docs/UserGuide.md (hunk 9 above)
Comment (Bluntsord) [reply docs/UserGuide.md:37]: Think should be Additionally links are present at the end of every section that bring u back .....

File: docs/UserGuide.md (hunk 10 above)
Comment (Bluntsord) [reply docs/UserGuide.md:38]: Noted and amended

File: docs/UserGuide.md (hunk 8 above)
Comment (Bluntsord) [reply docs/UserGuide.md]: noted and amend

File: docs/UserGuide.md (hunk 9 above)
Comment (sheehui) [reply docs/UserGuide.md:37]: can also

File: docs/UserGuide.md (hunk 7 above)
Comment (Bluntsord) [reply docs/UserGuide.md]: noted and amended

File: docs/UserGuide.md (hunk 6 above)
Comment (Bluntsord) [reply docs/UserGuide.md]: noted and amended.

File: docs/UserGuide.md (hunk 4 above)
Comment (bharathcs) [reply docs/UserGuide.md]: we are **used to**

File: docs/UserGuide.md [hunk 11]
```diff
@@ -34,12 +34,12 @@ As most applications on the market are not CLI-based, this may come across
 as new to some users. We strongly recommend that first-time users read through the user guide to gain a basic understanding
 of how to use our app. We have also included links to navigate around the user guide to allow easier access
 of related information. Some tips and tricks, as well as cautions are included throughout the guide to
-help you get more out of our application and avoid certain common dangers. Also, links back to the top of the page
-like below are present at the end of every section. To allow for easier navigation 
-around the user guide, we suggest using the table of contents as well as these links to quickly
-find and access the information needed. 
+help you get more out of our application. Additionally, links back to the top of the page
+like below are present at the end of every section that bring you back to the start of our user guide.
```
Comment (sheehui) [inline docs/UserGuide.md:38]: i think there's repetition eh. "Additionally, links that bring you back to the start of our user guide are present at the end of every section." is sufficient

File: docs/UserGuide.md (hunk 4 above)
Comment (sheehui) [reply docs/UserGuide.md]: ohh are you going to put the client details in?

File: docs/UserGuide.md [hunk 12]
```diff
@@ -568,7 +568,8 @@ This marks the end of the command summary section. Click
 
 ## 7. Glossary
 
-* **CLI**: Abbreviation for command line interface, where commands are processed in the form of text.
+* **CLI**: Abbreviation for command line interface, where commands are processed in the form of text
+as opposed to graphical user interfaces(mouse based applications).
```
Comment (sheehui) [inline docs/UserGuide.md:572]: mouse-based

**Review by sheehui — APPROVED** [review]
looks good, thanks!

**Pull request merged**